webdriver/tests/state/get_element_property.py

```diff
@@ -161,9 +161,9 @@ def test_element(session):
     session.url = inline("<input type=checkbox>")
     element = session.find.css("input", all=False)
     element.click()
-    assert None == session.execute_script("return arguments[0].hasAttribute('checked')",
-                                          args=[element])
-    result = session.transport.send("GET", "session/{session_id}/element/{element_id}/property/id"
+    assert False == session.execute_script("return arguments[0].hasAttribute('checked')",
+                                           args=[element])
```

Comment:
This is fine, but do you mean also replacing `==` with `is`, and converting the args list to a tuple? This statement should read:

```py
assert session.execute_script("return arguments[0].hasAttribute('checked')", args=(element,)) is False
```

Reply:
Gladly. Done in next commit.